setup.rst

```diff
@@ -281,6 +281,19 @@ create new projects. If you use Composer, you need to tell the exact version:
 
     $ composer create-project symfony/skeleton:"6.4.*" my_project_directory
 
+With an already existing project, you can restrict Symfony packages to one
+LTS version by :doc:`using Symfony Flex in your project </setup/flex>`
```

Comment:
```suggestion
With an already existing project, you can restrict Symfony packages to one
specific version by :doc:`using Symfony Flex in your project </setup/flex>`
```
it can be 5.3 as well

Reply:
While this is true, is it a good use case? Non-LTS versions are supported for 6 months. I don’t see the point of locking to something like 5.3 or 7.1.